test/api/v3/unit/libs/payments.test.js

```diff
@@ -55,6 +61,21 @@ describe('payments/index', () => {
       expect(difference - 60).to.be.lessThan(3); // the difference is approximately two months, +/- 2 days
     });
 
+    it('data.nextBill is undefined', () => {
```

Comment:
For instance, the test you created above, `it('sets subscription length'`, is great. It reads like a sentence and tells me exactly what you expect to happen. `it('data.nextBill is undefined'` does not do either of those things.